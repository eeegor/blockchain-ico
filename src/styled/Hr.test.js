import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Hr } from '.';

describe('Hr', () => {
	test('renders default header', () => {
		const tree = renderer.create(<Hr>Content</Hr>).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
