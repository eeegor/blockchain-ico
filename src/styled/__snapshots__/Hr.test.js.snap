// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hr renders default header 1`] = `
.c0 {
  width: 100%;
  margin: 26px auto 28px auto;
  outline: 0;
  border: 0;
  border-top: 1px solid #3332;
  max-width: 1006px;
}

<hr
  className="c0"
>
  Content
</hr>
`;
